ui/app/components/app/account-menu/index.scss

```diff
@@ -161,8 +161,8 @@
   }
 
   &__action {
-    font-size: 16px;
-    line-height: 18px;
+    @include Paragraph;
```

Comment:
The line height differs here, but I suspect this class is unused regardless. I can't find a reference to it anyway.